tox.ini

```diff
@@ -58,10 +68,15 @@ commands=
     sphinx-build -W docs/source build/docs
 
 [testenv:jwst]
+setenv=
+    CRDS_SERVER_URL = https://jwst-crds.stsci.edu
+    CRDS_PATH = ~/crds_cache
```

Comment:
I tried running this on my laptop and it created a directory named `~` in the repo root.  Maybe we ought to specify a path relative to one of the tox directory variables?

Reply:
That's a good idea.  The other option would be to define these at the job level in the CI workflow and have them pass through to the jwst job.  I wonder if that's the better alternative?